OpenSim/Sandbox/HumphreysImplicitDynamics/forwardImplicitTest.cpp

```diff
@@ -0,0 +1,368 @@
+/* -------------------------------------------------------------------------- *
+ *                             OpenSim:  forwardImplicitTest.cpp                             *
+ * -------------------------------------------------------------------------- *
+ * The OpenSim API is a toolkit for musculoskeletal modeling and simulation.  *
+ * See http://opensim.stanford.edu and the NOTICE file for more information.  *
+ * OpenSim is developed at Stanford University and supported by the US        *
+ * National Institutes of Health (U54 GM072970, R24 HD065690) and by DARPA    *
+ * through the Warrior Web program.                                           *
+ *                                                                            *
+ * Copyright (c) 2005-2012 Stanford University and the Authors                *
+ *                                                                            *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may    *
+ * not use this file except in compliance with the License. You may obtain a  *
+ * copy of the License at http://www.apache.org/licenses/LICENSE-2.0.         *
+ *                                                                            *
+ * Unless required by applicable law or agreed to in writing, software        *
+ * distributed under the License is distributed on an "AS IS" BASIS,          *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   *
+ * See the License for the specific language governing permissions and        *
+ * limitations under the License.                                             *
+ * -------------------------------------------------------------------------- */
+
+/* 
+ *  Below is an example of an OpenSim application that provides its own 
+ *  main() routine.  
+ */
```

Comment:
I broke out the dynamics test case to run it separately.  So this will become one of the test cases.  As far as the conventional test cases, I do plan to add those, but they rely on the cache variables which I will need to implement.  I am about done with that file and we can talk about if I should go ahead with the cache variables or what if anything we want to do different than the Millard Muscle.